### PR TITLE
Add support for `minimal_valve_position` parameter for Shelly Valve

### PR DESCRIPTION
**Pull request by bieniu**
Closes https://github.com/bieniu/ha-shellies-discovery/issues/498

### CODE REVIEW

**Comment by GergelyT** [comment]
I'd suggest the following formula if for any reason the valve goes below the minimal pos:
{{%elif value_json.thermostats.0.pos<=min_pos}%}}idle

Is this the same minimal_valve_position parameter, that was already supported ? I mean this update will not introduce the Minimal Valve Postition to Report Open parameter?

**Comment by bieniu** [comment]
Honestly, I don't like this change, we have to add an additional parameter, which is actually available but in a different payload. This is just stupid. That's why I opened a feature request to add `valve_min_percent`/`valve_min_report` to the payload from the `info` topic. We'll see what the Shelly team's response will be.

**Comment by GergelyT** [comment]
I totally understand you and if it is not a critical issue, you can cancel my request. minimal_valve_position itself was already mapped in your code and this is why I thought about stepping further. But I didn't see the issues coming from different payloads.

File: info.md [hunk 1]
```diff
@@ -244,9 +244,10 @@ key | optional | type | default | possible values | description
 `ext-switch` | True | boolean | `false` | `true`, `false` | presence of external switch
 `force_update_sensors` | True | boolean | `false` | `true`, `false` | [force update](https://www.home-assistant.io/integrations/sensor.mqtt/#force_update) for sensors
 `powered` | True | string | `battery` | `ac`, `battery` | `ac` or `battery` powered for Shelly H&T, Motion, Sense and Button1
-`expire_after` | True | integer | 51840 | | [expire after](https://www.home-assistant.io/integrations/binary_sensor.mqtt/#expire_after) for battery powered sensors in seconds
+`expire_after` | True | integer | `51840`` | | [expire after](https://www.home-assistant.io/integrations/binary_sensor.mqtt/#expire_after) for battery powered sensors in seconds
```
Comment (bieniu) [inline info.md:247]:
```suggestion
`expire_after` | True | integer | `51840` | | [expire after](https://www.home-assistant.io/integrations/binary_sensor.mqtt/#expire_after) for battery powered sensors in seconds
```

**Comment by bieniu** [comment]
No response from Shelly team so I will merge this change.

**Pull request merged**